pkg/process/process_test.go

```diff
@@ -0,0 +1,99 @@
+package process
```

Comment:
Looks like we need only public names, so we can use process_test package. This will be better tests since the test code will be the same as real code using this package.